standalone.js

```diff
@@ -8,40 +8,35 @@ const getSupportInfo = require("./src/main/support").getSupportInfo;
 const internalPlugins = [
   require("./src/language-js"),
   require("./src/language-css"),
-  require("./src/language-handlebars"),
+  // require("./src/language-handlebars"),
   require("./src/language-graphql"),
   require("./src/language-markdown"),
-  require("./src/language-html"),
+  // require("./src/language-html"),
   require("./src/language-vue")
 ];
-const externalPlugins = {};
+
+const isArray =
+  Array.isArray ||
+  function(arr) {
+    return Object.prototype.toString.call(arr) === "[object Array]";
+  };
 
 // Luckily `opts` is always the 2nd argument
 function withPlugins(fn) {
   return function() {
     const args = Array.from(arguments);
-    const opts = args[1] || {};
-    args[1] = Object.assign({}, opts, {
-      plugins: internalPlugins.concat(
-        (opts.plugins || [])
-          .map(
-            plugin =>
-              typeof plugin === "string" ? externalPlugins[plugin] : plugin
-          )
-          .filter(Boolean)
-      )
+    let plugins = (args[1] && args[1].plugins) || [];
+    if (!isArray(plugins)) {
+      plugins = Object.values(plugins);
+    }
```

Comment:
@azz We already expose the plugins to `global.prettierPlugins` so this will accept both `{ plugins: { string: [object] } }` and `{ plugins: [object] }`

Reply:
Makes sense.